enterprise/frontend/src/metabase-enterprise/api/table-data-edit.ts

```diff
@@ -79,6 +81,27 @@ export const tableDataEditApi = EnterpriseApi.injectEndpoints({
         actions: Array<WritebackAction | TableAction>;
       }) => response?.actions,
     }),
+    executeAction: builder.mutation<
+      TableExecuteActionResponse,
+      TableExecuteActionRequest
+    >({
+      query: ({ actionId, parameters }) => ({
+        method: "POST",
+        url: `/api/ee/data-editing/action/v2/execute`,
+        body: {
+          input: parameters,
+          // Here we pass a dummy table id, because the BE doesn't allow scope to be optional,
+          // but it's not actually used for this case.
+          scope: { "table-id": 1 },
```

Comment:
I think that scope should be added as an argument for `executeAction`. Even though we don't (currently) need scope for legacy "model actions", they are used for v2 actions (e.g. the actions grouped by table in the new picker). 

As far as I can tell this function is always getting called either from the Table page or from a Dashcard for an Editable, is that right? In that case we should pass the corresponding `table-id` or `dashcard-id` parameter as the scope - exactly like the scopes we send for data edits.